src/coreclr/src/vm/eventpipesession.h

```diff
@@ -96,6 +96,8 @@ class EventPipeSession
 
     void DisableIpcStreamingThread();
 
+    bool m_paused;
```

Comment:
Can we document why this variable is not protected by a lock and its specific use case?